### PR TITLE
Simplified Floor Tiles

### PR DESCRIPTION
**Pull request by jilfranco-unity**
### Description (*)
This PR is part of the art improvements for GDC epic. The ~750 tiny floor tiles have been replaced with a mix of ~150 larger and small floor tiles. Replacing the hundreds of small tiles with larger tiles reduces scene complexity (making it easier to manage and modify the level), reduces vert/tri count (aiding performance), and makes lighting bakes cleaner/faster (by reducing the amount of objects that need to bake, and reducing the amount of places there can be seams).

Prefabs for the new pieces have been created

Prefabs for Entrance, Transition Area, and Boss Room have been changed to use these new tiles

Lighting has been rebaked

### Issue Number(s) (*)
[Jira ticket here](https://jira.unity3d.com/browse/MTT-2758)

### CODE REVIEW

**Comment by SamuelBellomo** [comment]
@jilfranco-unity 
Is this just for GDC or both GDC and develop?
Also watch out, this points to develop at the moment.

**Comment by jilfranco-unity** [comment]
> 

This would be for both! I think it's good to have this change overall. At which branch should it point if I want the change for both?

**Comment by SamuelBellomo** [comment]
> > 
> 
> This would be for both! I think it's good to have this change overall. At which branch should it point if I want the change for both?

@jilfranco-unity this is fine, but we'll need to cherrypick your squashed commit once its merged to develop. Hopefully that's something you can do (in case there are conflicts). We can do it together if you need help.

File: Assets/BossRoom/Prefabs/Dungeon/Dungeon Pieces/env_floor3.prefab [hunk 1]
```diff
@@ -92,7 +92,7 @@ BoxCollider:
   m_GameObject: {fileID: 4371491365999748502}
   m_Material: {fileID: 0}
   m_IsTrigger: 0
-  m_Enabled: 1
```
Comment (fernando-cortez) [inline Assets/BossRoom/Prefabs/Dungeon/Dungeon Pieces/env_floor3.prefab:95]: So from what I remember, the BoxCollider component on these floor tile prefabs should be enabled (and in-scene as well) to bake the NavMesh. A script on awake turns these components off, and the big level plane is used instead for the raycasts from click/touch.

File: Assets/BossRoom/Prefabs/Dungeon/Dungeon Pieces/env_floor7.prefab [hunk 2]
```diff
@@ -0,0 +1,112 @@
+%YAML 1.1
```
Comment (fernando-cortez) [inline Assets/BossRoom/Prefabs/Dungeon/Dungeon Pieces/env_floor7.prefab:1]: So I'm noticing a considerable amount of smaller floor pieces on top of the newer big floor pieces.

These are just accent pieces, right? I'm curious how it'd look like without them and just rely on the bigger pieces, to double down on the optimization we're gaining.

Comment (jilfranco-unity) [reply Assets/BossRoom/Prefabs/Dungeon/Dungeon Pieces/env_floor7.prefab:1]: They are just accent pieces, yeah! I think just having the same texture repeated everywhere won't look very good, but let me try modifying the UVs of the big pieces so they have more texture variation!

**Comment by jilfranco-unity** [comment]
Removed accent floor 2 & floor 3 pieces to further simplify the floor down to ~60 pieces. Edited the UVs of the bigger floor tiles to break up repeating texture with the texture that was on floor 2

**Review by SamuelBellomo — DISMISSED** [review]

**Review by fernando-cortez — COMMENTED** [review]
Can you push the changes to the floor Colliders to the prefabs themselves as well?

**Review by SamuelBellomo — DISMISSED** [review]

**Review by SamuelBellomo — APPROVED** [review]

**Review by fernando-cortez — APPROVED** [review]

**Pull request merged**